tests/panels/test_sql.py

```diff
@@ -225,14 +225,32 @@ def test_json_param_conversion(self):
         connection.vendor == "postgresql", "Test valid only on PostgreSQL"
     )
     def test_tuple_param_conversion(self):
+        """
+        For psycopg3 You cannot use IN %s with a tuple
+        https://www.psycopg.org/psycopg3/docs/basic/from_pg2.html#you-cannot-use-in-s-with-a-tuple
+        """
+        is_psycopg3 = True
+        try:
+            import psycopg  # noqa: F401
+        except ImportError:
+            is_psycopg3 = False
+
         self.assertEqual(len(self.panel._queries), 0)
 
-        list(
-            PostgresJSON.objects.raw(
-                "SELECT * FROM tests_postgresjson WHERE field ->> 'key' IN %s",
-                [("a", "b'")],
+        if is_psycopg3:
+            list(
+                PostgresJSON.objects.raw(
+                    "SELECT * FROM tests_postgresjson WHERE field ->> 'key' = ANY(%s)",
+                    [["a", "b'"]],
+                )
+            )
```

Comment:
Sorry, we had a misunderstanding. This block does not fulfill the purpose of the test. If psycopg3 doesn't support tuples as a parameter, this test should fail expectedly. However, it doesn't look like there's a `expectedFailureIf` decorator, so maybe we update the `skipUnless` to include confirming it's psycopg2?

Reply:
was not sure how to check if we have psycopg3

so i relied on the method used with django to detect psycopg version
https://github.com/django/django/blob/main/django/db/backends/postgresql/base.py#L24

and combined it with `self.skipTest`

let me know if there is a better way to skip the test

Reply:
@nofalx can you enable maintainers to edit the PR? I'd like to push a change. It should be on the right side near the notifications button. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork